Lesson3/IFileHandling.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Collections.Generic;
+
+namespace Project_Console
+{
+    public interface IFileHandling
+    {
+        void DisplayData(List<string> parsedData);
+
+        string GetFilePath(string clArgumentNameValue);
```

Comment:
`Purposeful Code`: Where is this used through your interface?

Reply:
used in the GetParsedData method:
GetParsedData(string clArgumentNameValue) => ParseFileData(GetFilePath(clArgumentNameValue)).ToArray();